couchdb-setup.sh

```diff
@@ -43,6 +43,8 @@ curl -X PUT $COUCHURL/_global_changes
 curl -X PUT $COUCHURL/meetups
 curl -X PUT $COUCHURL/resources
 curl -X PUT $COUCHURL/courses
+curl -X PUT $COUCHURL/nations
+curl -X PUT $COUCHURL/community
```

Comment:
change to communityregistrationrequests

Reply:
changed to communityregistrationrequests